framework/Optimizers/FiniteDifference.py

```diff
@@ -37,13 +37,16 @@
 from utils import mathUtils,randomUtils
 #Internal Modules End--------------------------------------------------------------------------------
 
-class FiniteDifferenceGradientOptimizer(SPSA):
+class FiniteDifference(SPSA):
```

Comment:
XSD did not get changed, + a conversion script should be added

Reply:
changed XSD, conversion script is "Opt_FD_name.py".